site/content/how-to/data-plane-configuration.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Data plane configuration"
+weight: 400
+toc: true
+docs: "DOCS-000"
+---
+
+Learn how to dynamically update the NGINX Gateway Fabric global data plane configuration.
+
+## Overview
+
+NGINX Gateway Fabric can dynamically update the global data plane configuration without restarting. The data plane configuration is a global configuration for nginx that has options that are not available using the standard Gateway API resources. This includes such things as setting an OpenTelemetry collector config, disabling http2, or changing the IP family.
+
+The data plane configuration is stored in the NginxProxy custom resource, which is a cluster-scoped resource that is attached to the `nginx` GatewayClass.
+
+By default, the NginxProxy resource is not created when installing NGINX Gateway Fabric. However, you can set configuration options in the `nginx.config` Helm values, and the resource will be created and attached when NGINX Gateway Fabric is installed using Helm. You can also [manually create and attach](#manually-creating-the-configuration) the resource after NGINX Gateway Fabric is already installed.
+
+If installed using the Helm chart, the NginxProxy resource is named `<release-name>-proxy-config`.
+
+**For a full list of configuration options that can be set, see the `NginxProxy spec` in the [API reference]({{< relref "reference/api.md" >}}).**
```

Comment:
```suggestion
**For a full list of configuration options that can be set, see the `NginxProxy spec` in the [API reference]({{< relref "reference/api.md" >}}).**

This notice is repeated a few times: it could be a good candidate for an include for the sake of re-use.
```